ml_engine/online_prediction/predict.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2016 Google Inc. All Rights Reserved. Licensed under the Apache
+# License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+"""Examples of using the Cloud ML Engine's online prediction service."""
+from __future__ import print_function
+# [START import_libraries]
+import googleapiclient.discovery
+# [END import_libraries]
+
+
+# [START authenticating]
+def get_ml_engine_service():
+    """Create the ML Engine service object.
+    To authenticate set the environment variable
+    GOOGLE_APPLICATION_CREDENTIALS=<path_to_service_account_file>
+    """
+    return googleapiclient.discovery.build('ml', 'v1beta1')
+# [END authenticating]
+
+
+# [START predict_json]
+def predict_json(project, model, instances, version=None):
+    """Send data instances to a deployed model for prediction
+    Args:
+        project: str, project where the Cloud ML Engine Model is deployed.
+        model: str, model name.
+        instances: [dict], dictionaries from string keys defined by the model
+        to data.
+        version: [optional] str, version of the model to target.
+    Returns:
+        A dictionary of prediction results defined by the model.
```

Comment:
We generally encourage snippets to be simple enough not to require this, but I understand if that's not reasonable here. If you're going to go full docstring, follow Napoleon style:

```
Args:
    project (str): ...
    model (str): ...
    instances (Mapping[ str, dict ]): ...
    version (str): optional ...

Returns:
   Mapping [str, ...] : ...

```